src/libtilt/grids/central_slice_grid.py

```diff
@@ -43,7 +43,7 @@ def rotated_central_slice_grid(
         device=device,
     )  # (h, w, 3)
     if rotation_matrix_zyx is False:
-        grid = torch.flip(grid, dims=(-1,))
+        grid = torch.flip(grid, dims=(-1,)) #TODO: This operation is slow since it is copying the full tensor
```

Comment:
```suggestion
        grid = torch.flip(grid, dims=(-1,))
```

totally agree this is super frustrating, I'm sure you came across this but negative strides aren't supported in torch. I implemented support for matrices which work directly on zyx coords for this reason :-)

let's remove this todo and make an issue to track?

Reply:
#55 opened